src/com/aca/vardanmanasaryan/homework1/management/Spygadgets.java

```diff
@@ -0,0 +1,31 @@
+package com.aca.vardanmanasaryan.homework1.management;
+
+public abstract class Spygadgets implements SaleItem {
```

Comment:
Spy gadget for instance pen or usb flash can't sale item, this is wrong functionality